.circleci/config.yml.in

```diff
@@ -309,6 +309,9 @@ jobs:
       - pip_install:
           args: mypy
           descr: Install Python type check utilities
+      - run:
+          name: Install library stub packages
+          command: mypy --install-types
```

Comment:
Although we currently only need [types-requests](https://pypi.org/project/types-requests/), this is a convenience command to autodetect all missing library stubs and install them.